### PR TITLE
UI changes for Kafka Serialization Format selection

### PR DESCRIPTION
**Pull request by krithika369**
This PR adds an option to select the message serialization format, from one of 'JSON' and 'Protobuf', for Kafka Result Logging.

The 2 PRs below added support for Protobuf serialization of Kafka messages:
* https://github.com/gojek/turing/pull/31 (Router)
* https://github.com/gojek/turing/pull/32 (API)

This PR updates the UI with the corresponding changes and is the last of them to complete the Protobuf support for Kafka logger.

### Modifications

* `ui/src/router/components/configuration/components/LoggingConfigSection.js` - Display Serialization Format if Kafka logging has been selected
* `ui/src/router/components/form/components/outcome_config/kafka/KafkaConfigPanel.js` - Add `EuiSuperSelect` to choose the serialization formation
* `ui/src/router/components/form/validation/schema.js` - Define the oneOf validation for the new property
* `ui/src/config.js` - Process `REACT_APP_RESULT_LOG_PROTO_URL`. This will be used in the Edit Router form, to link to the `.proto` file in the Kafka Config panel.

Screenshot 2020-12-08 at 11 21 06 PM

### CODE REVIEW

File: ui/src/router/activity_log/events_list/EventsList.scss [hunk 1]
```diff
@@ -1,3 +1,7 @@
 .activityLogEvent {
+  // white-space: pre-wrap renders \n in the text as a new line
   white-space: pre-wrap;
+  // word-break: break-word will prevent the possibility of horizontal scroll
+  // on the text by always breaking when the text overflows
+  word-break: break-word;
```
Comment (krithika369) [inline ui/src/router/activity_log/events_list/EventsList.scss:6]: This change relates to a previous PR: https://github.com/gojek/turing/pull/29

File: ui/src/router/components/form/components/outcome_config/kafka/KafkaConfigPanel.js [hunk 2]
```diff
@@ -1,15 +1,39 @@
...
+const logSerializationFormatOptions = [
+  {
+    value: "json",
+    inputDisplay: "JSON"
+  },
+  {
+    value: "protobuf",
+    inputDisplay: "Protobuf",
+    helpText: "TuringResultLog.proto",
+    helpUrl: resultLoggingConfig.protoUrl
```
Comment (romanwozniak) [inline ui/src/router/components/form/components/outcome_config/kafka/KafkaConfigPanel.js:24]: Would it be a better idea to make `helpText` one of `string | ReactNode`? So you can specify the entire logic here as:
 ```js
helpText: resultLoggingConfig.protoUrl 
  ? (
    <EuiLink href={resultLoggingConfig.protoUrl} target="_blank" external>
            TuringResultLog.proto
    </EuiLink>
  ) : ""
```

and then use it at line 91 as: 
```js
   helpText={ (selectedFormat || {}).helpText }
```

File: ui/src/services/router/TuringRouter.js [hunk 3]
```diff
@@ -49,7 +49,8 @@ export class TuringRouter {
         },
         kafka_config: {
           brokers: "",
-          topic: ""
+          topic: "",
+          serialization_format: ""
```
Comment (romanwozniak) [inline ui/src/services/router/TuringRouter.js:53]: Do we want to be opinionated about the default serialization format? Shall we set it as `Protobuf` and save user an extra click (actually, two)?

**Review by romanwozniak — APPROVED** [review]
I've added few suggestions, but it LGTM

**Pull request merged**